### PR TITLE
Added mining with G1 ASIC steps to extra_documents

### PR DESCRIPTION
**Pull request by cekickafa**
Added tutorial images to docs/assets/images/g1-mini-ASIC folder.

### CODE REVIEW

File: docs/wiki/extra-documents/how-to-mine-with-G1-mini-ASIC.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+Download the file [Showip.exe](https://gameway.fun/output/api/ipollo/download?fileName=Showip.rar).
+
+Right click the “Show IP.exe” and run it as Administrator.
+
+Press the “IP Report” button on the server for 3-5 seconds and release it.
+
+<img alt="grin tutorial image 1" width="400px" src="../../assets/images/g1-mini-ASIC/G1-tut-2.png" />
+
+The server IP address will be feedback as shown below:
```
Comment (phyro) [inline docs/wiki/extra-documents/how-to-mine-with-G1-mini-ASIC.md:26]: `The server IP address will be feedback as shown below` -> `The server IP address will be shown as can be seen below:`

File: docs/wiki/extra-documents/how-to-mine-with-G1-mini-ASIC.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+Pool3：stratum+tcp://grin.always.vip:3344
+
+Pool4：stratum+tcp://grin.us.always.vip:3344
+
+*For Decentralization and healthy Network, choose other pools please.
+
+Pool Worker：According to the“sub-accounts”set in the corresponding mining pool account, if the sub-account is ipollo123, the worker should be ipollo123.xxx, the number suffixed used to
+
+distinguish the server.
```
Comment (phyro) [inline docs/wiki/extra-documents/how-to-mine-with-G1-mini-ASIC.md:75]: It seems like a newline slipped here (you can see it breaks the statement here https://github.com/mimblewimble/docs/blob/1a283cea676814fb01a5c8ddc1f54e6323a137a7/docs/wiki/extra-documents/how-to-mine-with-G1-mini-ASIC.md).

**Comment by phyro** [comment]
This looks great! Let's take this as an opportunity to get someone that is mining with G1 Mini involved to review the steps as I have absolutely no idea whether any of this is accurate :+1: It's also necessary to get more people involved in confirming the changes.

P.S. The `Check Markdown links / markdown-link-check (pull_request)` is not related to your PR and we'll have to go through these on a separate occasion.

File: docs/wiki/extra-documents/how-to-mine-with-G1-mini-ASIC.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+Caution: The server and PC must be in the same network segment, otherwise the IP information returned by the server cannot be received.
+
+
+<hr/>
+<br/>
+
+## Find the server IP address.
+
+Download the file [Showip.exe](https://gameway.fun/output/api/ipollo/download?fileName=Showip.rar).
```
[the remark this answers is not in the document]
Comment (cekickafa) [reply docs/wiki/extra-documents/how-to-mine-with-G1-mini-ASIC.md:18]: Official Ipollo file,taken from their download.

File: docs/wiki/extra-documents/how-to-mine-with-G1-mini-ASIC.md (hunk 2 above)
Comment (cekickafa) [reply docs/wiki/extra-documents/how-to-mine-with-G1-mini-ASIC.md]: I removed blank line.

File: docs/wiki/extra-documents/how-to-mine-with-G1-mini-ASIC.md (hunk 1 above)
Comment (cekickafa) [reply docs/wiki/extra-documents/how-to-mine-with-G1-mini-ASIC.md]: Corrected.

**Pull request merged**